server/actions/Event.ts

```diff
@@ -20,6 +20,20 @@ export const getEvent = async function (id: string) {
     return event;
 };
 
+/**
+ * @returns All events.
+ */
+export const getEvents = async function () {
+    await mongoDB();
+
+    const events = (await EventSchema.find({})) as Array<Event>;
+    if (events == null) {
```

Comment:
Can you change this to `if (!events || !events.length)`? `!events` is the same thing as you have right now. But I think we'd also need a check the array length here.

Reply:
Done!